### ISSUE TITLE
Old subprojects still have _version.pys laying around

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8219]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#8219|
|Type|release blocker: regression|
|Created|2016-02-25 00:35:07Z|
|Branch|https://github.com/twisted/twisted/tree/old-versionpy-8219|

These are incorrect, the __version__ should come from the main _version.py, not this one.

<details><summary>Searchable metadata</summary>

```
trac-id__8219 8219
type__release_blocker__regression release blocker: regression
reporter__hawkowl hawkowl
priority__normal normal
milestone__Twisted_16_0 Twisted 16.0
branch__branches_old_versionpy_8219 branches/old-versionpy-8219
branch_author__hawkowl hawkowl
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1456360507887601 1456360507887601
changetime__1456629369767724 1456629369767724
version__None None
owner__hawkowl hawkowl

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_1"></a>|@hawkowl commented|
|-|-|

(In [46843]) Branching to old-versionpy-8219.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_2"></a>|@hawkowl commented|
|-|-|

Okay, happy with this. Please review.

**Comment by twisted-trac** [comment]
|[<img alt="mithrandi's avatar" src="https://avatars.githubusercontent.com/u/856430?s=50" width="50" height="50">](https://github.com/mithrandi)<a name="note_3"></a>|@mithrandi commented|
|-|-|

Looks good, please merge :)

**Comment by twisted-trac** [comment]
|[<img alt="mithrandi's avatar" src="https://avatars.githubusercontent.com/u/856430?s=50" width="50" height="50">](https://github.com/mithrandi)|@mithrandi set owner to @hawkowl|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="mithrandi's avatar" src="https://avatars.githubusercontent.com/u/856430?s=50" width="50" height="50">](https://github.com/mithrandi)<a name="note_5"></a>|@mithrandi set status to `closed`|
|-|-|

(In [46855]) Merge old-versionpy-8219: Remove old _version.py files.

Fixes: [#8219](https://github.com/twisted/twisted/issues/8219)
Author: hawkowl
Reviewer: mithrandi

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_6"></a>|@hawkowl commented|
|-|-|

(In [46861]) Merge old-versionpy-8219: Remove old _version.py files.

Fixes: [#8219](https://github.com/twisted/twisted/issues/8219)
Author: hawkowl
Reviewer: mithrandi
